hwangsae/recorder.h

```diff
@@ -52,11 +52,14 @@ void                    hwangsae_recorder_set_max_size_bytes
 guint64                 hwangsae_recorder_get_max_size_bytes
                                                        (HwangsaeRecorder * self);
 
-void                    hwangsae_recorder_start_recording
+gint64                  hwangsae_recorder_get_recording_id
+                                                       (HwangsaeRecorder * self);
+
+gint64                  hwangsae_recorder_start_recording
                                                        (HwangsaeRecorder * self,
                                                         const gchar * uri);
 
-void                    hwangsae_recorder_stop_recording
+gint64                  hwangsae_recorder_stop_recording
```

Comment:
I think this API is a bit strange. Normally, when we stop recording, we should know what recording we want to stop.
So, I am not sure what we can do with returned `record_id` by calling `hwangsae_recorder_stop_recording()`.

If we need to be sure what  `record-id` should be stopped, I think it's better to be `void hwangsae_recorder_stop_recording(HwangsaeRecorder *self, gint64 record_id, GError **error)`, and the last `error` parameter can be optional.

Reply:
The initial idea of returning the `record_id` was to match the REST API, which of course it is not needed, but sounded nice. I don't think adding `record_id` as parameter has much sense, as to check if this object handles the desired recording `hwangsae_recorder_get_recording_id` should be called.

Please confirm me in which way would you like to handle this.

Reply:
Sure, with this implementation, the parameters are not required as you said.

okay, let's remove record id even from REST API, and just keep the return as `void`.

Reply:
Sure, thanks for clarifying so quickly.